### ISSUE TITLE
OpenCV replacement

### ISSUE THREAD

**Issue by romainbrette** [romainbrette/holypipette#9]
Camera display and template matching currently use opencv, which can be a bit complicated to install. It could be good to have a simpler replacement. Possibly skimage for template matching?

**Comment by romainbrette** [comment]
Camera display now uses Qt. We still use opencv for image processing.

**Comment by romainbrette** [comment]
Opencv is used in:
* templatematching: could be replaced by scikit-image
* umanagercamera: convertScaleAbs; probably imageio would work
* microscope: just imwrite; could use imageio
* camera/camera: in RecordedVideoCamera: not used, and replaceable by imageio
* opencvcamera: can be kept as is
* interface/camera: used only for tracking (Hoang's code) and pipette contact detection. Both could go.

In summary, it is totally feasible to get rid of opencv. I suggest to do first the replacement with imageio, and in templatematching, use try import to select the right package.
